react-select/react-select.d.ts

```diff
@@ -19,48 +19,75 @@ declare namespace ReactSelect {
         clearableValue?: boolean;
     }
 
+    interface MenuRendererProps {
```

Comment:
See https://github.com/JedWatson/react-select/#effeciently-rendering-large-lists-with-windowing for a list of `menuRenderer` props